nautobot/utilities/utils.py

```diff
@@ -334,6 +337,25 @@ class NautobotFakeRequest:
     def __init__(self, _dict):
         self.__dict__ = _dict
 
+    def nautobot_serialize(self):
+        """
+        Serialize a json representation that is safe to pass to celery
+        """
+        data = copy.deepcopy(self.__dict__)
+        data["user"] = data["user"].pk
+        return data
```

Comment:
I'm realizing that this method is very similar to `to_csv()` except a dict. Excluding what I suggested earlier about attempting to actually leverage DRF model serializers for this pattern, this might be another opportunity to align these such as if we returned an ordered dict and had `to_csv()` return the values? Especially when you also consider `csv_headers` paired with `to_csv()`, that's a deconstructed ordered dict...

Reply:
As discussed above, we will revisit and hopefully remove this pattern later.